app/css/NameTag.css

```diff
@@ -50,21 +49,21 @@
 }
 
 .submit-btn:hover {
-  background-color: #d6624a;
+  background-color: #d6624a; /* No change needed */
 }
 
 .label-styling {
-  margin-left: 0px;
+  margin-left: 0; /* No change needed */
 }
 
 .controller-container {
-  margin-right: 10px;
-  margin-left: 0px;
-  margin-top: 10px;
-  margin-bottom: 10px;
-  padding: 0px 5px 0px 5px; 
```

Comment:
The margins don't necessarily need to be responsive.

Reply:
Ok fixing this.